packages/pytest-simcore/src/pytest_simcore/docker.py

```diff
@@ -43,10 +47,12 @@ async def _pause_container(
 
 
 @pytest.fixture
-async def paused_container() -> Callable[[str], AbstractAsyncContextManager[None]]:
+async def pause_container_in_context() -> Callable[
```

Comment:
suggestion: ```with_paused_container```

Reply:
that reads strange since it has double `with`.  I propose `paused_container_ctx`

Reply:
well to be honnest I would ditch anything that says context. types are supposed to be there for this.